app/models/version.rb

```diff
@@ -98,7 +99,10 @@ def estimated_hours
 
   # Returns the total reported time for this version
   def spent_hours
-    @spent_hours ||= TimeEntry.sum(:hours, include: :work_package, conditions: ["#{WorkPackage.table_name}.fixed_version_id = ?", id]).to_f
+    @spent_hours ||= TimeEntry.includes(:work_package)
+                     .where(["#{WorkPackage.table_name}.fixed_version_id = ?", id])
```

Comment:
This could be better written as:
`where(work_package: { fixed_version_id: 5 })`

Don't know if it is in the scope of this PR, though.